CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.4.1] - 2023/09/XX
```

Comment:
Let's make it a 0.5.0 as its not really a bug fix, but rather a behavior change.